src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const EVERY_DAY_COST = 40;
+  const DISCOUNT_AFTER_3_DAYS = 20;
+  const DISCOUNT_AFTER_7_DAYS = 50;
+
+  if (days <= 2) {
+    return days * EVERY_DAY_COST;
+  }
+
+  if (days <= 6) {
+    return days * EVERY_DAY_COST - DISCOUNT_AFTER_3_DAYS;
+  }
+
+  if (days >= 7) {
+    return days * EVERY_DAY_COST - DISCOUNT_AFTER_7_DAYS;
+  }
 }
```

Comment:
The function `calculateRentalCost` is named appropriately and uses constants instead of literals, which is good. However, the condition on line 19 should be simplified to an 'else' block because if the execution reaches that point, it means that `days` is necessarily 7 or more, given the previous conditions.